src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -1010,6 +1012,348 @@ static bool processAreaScope(Scope* scoP, BSONObj &areaQuery)
   return true;
 }
 
+/* ****************************************************************************
+*
+* qStringFilters -
+*
+* FIXME P9: this function currently abuses of the std::string() wrapping for char* in order to
+* make comparisons work. An smarter solution could be developed.
+*
+*/
+static void qStringFilters(std::string& in, std::vector<BSONObj> &filters)
```

Comment:
Why not put qStringFilters in its own file (qStringFilters,cpp/h)  ?
It is true that you can 'hide' the function in MongoGlobal (making it static) as it is only used in MongoGlobal ...
However, it wouldn't be a bad thing to have it in its own file, I think.

This is just a matter of taste, really.